app/models/document.rb

```diff
@@ -208,6 +214,12 @@ def save
   def publish!
     handle_remote_error do
       update_type = self.update_type || 'major'
+
+      if not_published?
+        self.change_note = "First published."
+        self.save
+      end
```

Comment:
I think we need to be careful with the `#not_published?` method. It returns true if the `publication_state` returned by Publishing API is not one of `live` or `redrafted`. The Publishing API can also return states of `republish` and `unpublish` I think.

I'm not sure what the right thing to do is, though. In the case of `republish` it's ambiguous as to whether the content item was previously published or not, i think. It might be safer to look through the `change_history` for the `First published.` item or maybe look at the `first_published_at` field which should always be set on a content item that has previously been published.